rust/config/mainnet3_config.json

```diff
@@ -1,6 +1,61 @@
 {
   "chains": {
+    "ancient8": {
+      "aggregationHookFactory": "0xEb9FcFDC9EfDC17c1EC5E1dc085B98485da213D6",
```

Comment:
pls do the renamings as done in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3471